service/iot-service-client/src/test/java/tests/unit/com/microsoft/azure/sdk/iot/service/FeedbackReceiverTest.java

```diff
@@ -34,7 +34,7 @@ public void constructor_input_hostName_null() throws Exception
         String deviceId = "xxx";
         IotHubServiceClientProtocol iotHubServiceClientProtocol = IotHubServiceClientProtocol.AMQPS;
         // Act
-        FeedbackReceiver feedbackReceiver = new FeedbackReceiver(hostName, userName, sasToken, iotHubServiceClientProtocol, deviceId);
+        FeedbackReceiver feedbackReceiver = new FeedbackReceiver(hostName, userName, sasToken, iotHubServiceClientProtocol);
```

Comment:
Can you say more about removing the `deviceId` parameter?

Reply:
This constructor was unused, and the test could have just been deleted. The test is fine for testing the same behavior on this other constructor though, so I kept it around

Reply:
More generally, the feedback receiver isn't tied to any particular device, so this constructor that took a deviceId never did anything with it to begin with. That was why we deprecated it in the first place. Feedback receivers can get messages for all devices